### PR TITLE
fix(job): a few bugs in scheduler ui

### LINKED ISSUE #1303: refactor(job-create): remove time from periodic jobs

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Can no longer set time for recurring jobs "from" "to"
- Plugin no longer crashes if trying to start with invalid cron syntax
- Added text to inform  that times are in UTC
- Styling so that "Simple" and "Advanced" button don't shift around

## Why is this pull request needed?
bugs

## Issues related to this change
closes #1303 
a few from #1252

### CODE REVIEW

File: packages/dm-core-plugins/src/job/JobControl.tsx [hunk 1]
```diff
@@ -75,8 +77,19 @@ export const JobControl = (props: IUIPlugin) => {
     if (!jobEntity) return
     jobEntity.triggeredBy =
       tokenData?.preferred_username || tokenData?.name || 'Anonymous'
-    // @ts-ignore
-    if (asCronJob) jobEntity.schedule = schedule
+    if (asCronJob) {
+      // @ts-ignore
```
Comment (awesthouse) [inline packages/dm-core-plugins/src/job/JobControl.tsx:81]: I agree with linting here - why are we updating state in a non-immutable way like this? Goes for both triggeredBy and schedule

File: packages/dm-core-plugins/src/job/SimpleJobControlButtons.tsx [hunk 2]
```diff
@@ -124,3 +134,17 @@ export const CompletedButton = (props: {
...
+export const ErrorButton = (props: { error: ErrorResponse }) => {
+  return (
+    <Button
+      style={{
+        backgroundColor: loadingColor,
+        cursor: 'not-allowed',
+      }}
+      variant='contained_icon'
+    >
+      <Icon data={error_outlined} style={{ color: 'white' }} />
```
Comment (awesthouse) [inline packages/dm-core-plugins/src/job/SimpleJobControlButtons.tsx:147]: <Icon /> also has a color prop if you want to use that 🙂 
`<Icon data={error_outlined} color="white" />`

**Review by awesthouse — APPROVED** [review]
Language and text seems much better and clearer 🙂 Looks good! And looks like we were already doing the state mutating before this pr, but I don't see a reason for doing it this way, would be good to fix it if there isn't a good reason

File: packages/dm-core-plugins/src/job/JobControl.tsx (hunk 1 above)
Comment (soofstad) [reply packages/dm-core-plugins/src/job/JobControl.tsx:81]: The linter did not complain, its typescript that is not able to see that the job is a recurring job.
However, no reason to mutate a state object neither :slightly_smiling_face:

**Pull request merged**